code/game/gamemodes/cult/cult_items.dm

```diff
@@ -72,6 +72,194 @@
 		if(is_servant_of_ratvar(C) && C.reagents)
 			C.reagents.add_reagent("heparin", 1)
 
+/obj/item/twohanded/required/cult_bastard
+	name = "bloody bastard sword"
+	desc = "An enormous sword used by Nar-Sien cultists to rapidly harvest the souls of non-believers."
+	w_class = WEIGHT_CLASS_HUGE
+	block_chance = 50
+	throwforce = 20
+	force = 35
+	armour_penetration = 45
+	throw_speed = 1
+	throw_range = 3
+	sharpness = IS_SHARP
+	light_color = "#ff0000"
+	attack_verb = list("cleaved", "slashed", "torn", "hacked", "ripped", "diced", "carved")
+	icon_state = "cultbastard"
+	item_state = "cultbastard"
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	lefthand_file = 'icons/mob/inhands/64x64_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/64x64_righthand.dmi'
+	inhand_x_dimension = 64
+	inhand_y_dimension = 64
+	actions_types = list()
+	flags_2 = SLOWS_WHILE_IN_HAND_2
+	var/datum/action/innate/cult/dash/jaunt
+	var/datum/action/innate/cult/spin2win/linked_action
+	var/spinning = FALSE
+	var/spin_cooldown = 250
+	var/list/shards = list()
+	var/dash_toggled = TRUE
+
+/obj/item/twohanded/required/cult_bastard/Initialize()
+	. = ..()
+	set_light(4)
+
+/obj/item/twohanded/required/cult_bastard/can_be_pulled(user)
+	return FALSE
+
+/obj/item/twohanded/required/cult_bastard/attack_self(mob/user)
+	dash_toggled = !dash_toggled
+	if(dash_toggled)
+		to_chat(loc, "<span class='notice'>You raise the [src] and prepare to jaunt with it.</span>")
+	else
+		to_chat(loc, "<span class='notice'>You lower the [src] and prepare to swing it normally.</span>")
+
+/obj/item/twohanded/required/cult_bastard/pickup(mob/living/user)
+	. = ..()
+	if(!iscultist(user))
+		if(!is_servant_of_ratvar(user))
+			to_chat(user, "<span class='cultlarge'>\"I wouldn't advise that.\"</span>")
+			to_chat(user, "<span class='warning'>An overwhelming sense of nausea overpowers you!</span>")
+			user.Dizzy(80)
+			user.Knockdown(30)
+			return
+		else
+			to_chat(user, "<span class='cultlarge'>\"One of Ratvar's toys is trying to play with things [user.p_they()] shouldn't. Cute.\"</span>")
+			to_chat(user, "<span class='userdanger'>A horrible force yanks at your arm!</span>")
+			user.emote("scream")
+			user.apply_damage(30, BRUTE, pick("l_arm", "r_arm"))
```

Comment:
Minor but, does this actually damage the arm you grabbed the sword with or does it pick a random one?

Reply:
It picks a random one.

Logic: You need both hands to lift/hold the sword.

Reality: I'm too lazy to look up how to tell which hand was used to touch something.